src/aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -121,7 +138,16 @@ def get_builder_from_protocol(
         if 'settings' in inputs['ph']:
             builder.ph['settings'] = orm.Dict(inputs['ph']['settings'])
         builder.clean_workdir = orm.Bool(inputs['clean_workdir'])
-        builder.ph['qpoints'] = qpoints
+
+        if 'qpoints' in inputs:
```

Comment:
@sphuber , @mbercx I wonder how to address something. I added this logic about qpoints, but i declareded the input as `spec.input('qpoints', valid_type=orm.KpointsData, required=False,
            help='An explicit qpoints list or mesh. Either this or `qpoints_distance` should to be provided.')` However I think it should be a list in order to not interfere with the logic from aiid-quantumespresso-ph ? Any opinions about this? Because i guess i should add test for this as well?

Reply:
Sorry for the slow response @AndresOrtegaGuerrero! I guess this comment is outdated? I realise now that this same logic is actually broken in the `PwBaseWorkChain`, will open a fix. 

https://github.com/aiidateam/aiida-quantumespresso/blob/bec6dd6b56b4cd3bbed3f3ab8fb97c7f7bdc0214/src/aiida_quantumespresso/workflows/pw/base.py#L225-L226